Source/objects.h

```diff
@@ -191,6 +191,17 @@ struct Object {
 		return IsAnyOf(_otype, _object_id::OBJ_CHEST1, _object_id::OBJ_CHEST2, _object_id::OBJ_CHEST3, _object_id::OBJ_TCHEST1, _object_id::OBJ_TCHEST2, _object_id::OBJ_TCHEST3);
 	}
 
+	[[nodiscard]] constexpr bool IsSarcophagus() const
+	{
+		return _otype == _object_id::OBJ_SARC;
+	}
+
+	[[nodiscard]] constexpr bool IsOpen() const
+	{
+		assert(IsChest() || IsSarcophagus());

```

Comment:
?

Reply:
Please elaborate on your question whose meaning I cannot get

Reply:
what's the point of this assert?